prime-dotnet-webapi/Migrations/20240702173818_AddSiteSubmission.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace Prime.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddSiteSubmission : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropCheckConstraint(
+                name: "CHK_Agreement_EitherPartyOrEnrollee",
+                table: "Agreement");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Discriminator",
+                table: "HealthAuthorityContact",
+                type: "character varying(55)",
+                maxLength: 55,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "text");
```

Comment:
Why is this part of Migration?

Reply:
Recreated the migration. It seems all unrelated changes are gone.